tests/end_to_end/helpers.h

```diff
@@ -99,103 +99,81 @@ typedef class _single_instance_hook
     ebpf_handle_t link_handle;
 } single_instance_hook_t;
 
+static ebpf_helper_function_prototype_t _ebpf_map_helper_function_prototype[] = {
+    {1,
+     "ebpf_map_lookup_element",
+     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+    {2,
+     "ebpf_map_update_element",
+     EBPF_RETURN_TYPE_INTEGER,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}},
+    {3,
+     "ebpf_map_delete_element",
+     EBPF_RETURN_TYPE_INTEGER,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}}};
+
+static ebpf_context_descriptor_t _ebpf_xdp_context_descriptor = {sizeof(xdp_md_t),
+                                                                 EBPF_OFFSET_OF(xdp_md_t, data),
+                                                                 EBPF_OFFSET_OF(xdp_md_t, data_end),
+                                                                 EBPF_OFFSET_OF(xdp_md_t, data_meta)};
+static ebpf_program_information_t _ebpf_xdp_program_information = {{"xdp",
+                                                                    &_ebpf_xdp_context_descriptor,
+                                                                    // clang-format off
```

Comment:
why is clang-format off needed here?

Reply:
As otherwise it was printing the guid as such:

```
    0xf1832a85,
    0x85d5,
    0x45b0,
    0x98,
    0xa0,
    0x70,
    0x69,
    0xd6,
    0x30,
    0x13,
    0xb0
```
I wanted it to show in the same format as 
uuidgen -s
INTERFACENAME = { /* 81ee4c5e-7599-4e72-b0c4-1be21b9b1841 */
    0x81ee4c5e,
    0x7599,
    0x4e72,
    {0xb0, 0xc4, 0x1b, 0xe2, 0x1b, 0x9b, 0x18, 0x41}
  };

Reply:
Fixed.